src/views/GifReferenceWidget.vue

```diff
@@ -22,38 +22,57 @@
 <template>
 	<div class="gif-reference">
 		<div class="gif-wrapper">
-			<div v-if="!isLoaded" class="loading-icon">
+			<div v-if="!isLoaded && gifsEnabled" class="loading-icon">
 				<NcLoadingIcon
 					:size="44"
 					:title="t('integration_giphy', 'Loading GIF')" />
 			</div>
-			<img v-show="isLoaded"
-				class="image"
-				:src="proxiedUrl"
-				@load="isLoaded = true">
-			<a v-show="isLoaded"
-				class="attribution"
-				target="_blank"
-				:title="poweredByTitle"
-				href="https://giphy.com">
-				<img :src="poweredByImgSrc"
-					:alt="poweredByTitle">
-			</a>
+			<NcButton :pressed.sync="gifsEnabled"
```

Comment:
What is the use of the pressed attr?

Reply:
Ah forgot to remove it from the example. It syncs multiple inputs (radio buttons or checkboxes). It is not required in this case.